literature/lipowski_1998_ionic_2875/lipowski_1998_ionic_2875.bib

```diff
@@ -0,0 +1,10 @@
+@article{lipowski_1998_ionic_2875,
```

Comment:
The first Author is called `Lipkowski`. 
Please change filenames and ID in the yaml files.

Reply:
```suggestion
@article{lipkowski_1998_ionic_2875,
```